www/getting-started.rst

```diff
@@ -2,151 +2,80 @@
 
-* *SciPy* contains additional routines needed in scientific work: for example,
-  routines for computing integrals numerically, solving differential equations,
-  optimization, and sparse matrices.
+`Jupyter <https://jupyter.org/>`_ notebooks combine code, markdown, and more in an interactive setting. They are an excellent tool for learning, collaborating, experimenting, or documenting. Notebooks can run on your local machine, and `MyBinder <https://mybinder.org/>`_ also serves Jupyter notebooks to the browser without the need for anything on the local computer. For example, `MyBinder Elegant Scipy <https://mybinder.org/v2/gh/elegant-scipy/notebooks/master?filepath=index.ipynb>`_ provides an interactive tutorial.
 
-* The matplotlib_ module produces high quality plots. With it you can turn your
-  data or your models into figures for presentations or articles. No need to do
-  the numerical work in one program, save the data, and plot it with another
-  program.
+Jupyter runs by calling to `IPython <https://ipython.org/>`_ behind the scenes, but IPython itself also acts as a standalone tool. A *command-line* of individual statements and returned values, IPython is useful for debugging and experimenting.
 
-* Using IPython_ makes interactive work easy. Data processing, exploration of
-  numerical models, trying out operations on-the-fly allows to go quickly from
-  an idea to a result. See the `IPython site <http://ipython.org/>`__ for many
-  examples.
+Editors for scripts run from minimal, like Window's Notepad or `Emacs <https://www.gnu.org/software/emacs/>`_ , to the fully-featured and customizable, like `Atom <https://atom.io/>`_ or `Visual Studio Code <https://code.visualstudio.com/>`_ . Jupyter and IPython work well for interacting with code and experimenting, but scripts, files with a `*.py` extension, form backends, infrastructure, or anything heavy-duty.  
 
-* There is a sizeable collection of both generic and
-  application-specific numerical and scientific code, written using
-  Python, NumPy and SciPy. Don't reinvent the wheel, there may already
-  be a pre-made solution for your problem. See
-  :doc:`topical-software` for a partial list.
+Hello SciPy 
+===========
 
-* As Python is a popular general-purpose programming language, it has
-  many advanced modules for building for example interactive
-  applications (see e.g. wxPython_ and Traits_) or web sites (see
-  e.g. Django_). Using SciPy with these is a quick way to build a
-  fully-fledged scientific application.
+Need to test if the packages got installed?  Type these lines at an IPython prompt, or save in a `*.py` file to execute::
 
-How to work with SciPy
-----------------------
-Python is a programming language, and there are several ways to
-approach it. There is no single program that you can start and that
-gives an integrated user experience.  Instead, there are several
-possible ways to work with Python.
+    import numpy as np
+    print("I like ",np.pi)
```

Comment:
:)

PEP8 nitpick: a space after the comma

Reply:
did that.